tests/poolTests.c

```diff
@@ -50,21 +51,45 @@ int testOrder(size_t numThreads, size_t queueSize) {
   return 0;
 }
 
+void waitFn(void *opaque) {
+  (void)opaque;
+  usleep(100);
```

Comment:
Note that we also have `UTIL__sleep()` and `UTIL_sleepMilli()` within `programs/util.h`
with the benefit of being portable on Windows and Posix.